vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -33,7 +33,7 @@ export class AcquisitionInvoker extends IAcquisitionInvoker {
     private noPowershellError = `powershell.exe is not discoverable on your system. Is PowerShell added to your PATH and correctly installed? Please visit: https://learn.microsoft.com/powershell/scripting/install/installing-powershell-on-windows.
 You will need to restart VS Code after these changes. If PowerShell is still not discoverable, try setting a custom existingDotnetPath following our instructions here: https://github.com/dotnet/vscode-dotnet-runtime/blob/main/Documentation/troubleshooting-runtime.md.`
 
-    constructor(extensionState: IExtensionState, eventStream: IEventStream, timeoutTime : number) {
+    constructor(extensionState: IExtensionState, eventStream: IEventStream, timeoutTime : number, installFolder?: string) {

```

Comment:
```suggestion
    constructor(extensionState: IExtensionState, eventStream: IEventStream, timeoutTime : number) {

```

It looks like `installFolder` is never used in the true class, so I would not include it in the constructor here